--- resize.go	2016-10-18 16:13:10.000000000 -0400
+++ resize-patched.go	2016-10-18 16:10:25.000000000 -0400
@@ -52,7 +52,7 @@
 	inHeight := int(image.Ysize)
 
 	// Infer the required operation based on the in/out image sizes for a coherent transformation
-	normalizeOperation(&o, inWidth, inHeight)
+	// normalizeOperation(&o, inWidth, inHeight)
 
 	// image calculations
 	factor := imageCalculations(&o, inWidth, inHeight)
